memory-bank/techContext.md

```
@@ -0,0 +1,101 @@
+pnpm mobile:android   # Run Android emulator
+pnpm mobile:ios       # Run iOS simulator
+pnpm build:android    # Build Android APK
+pnpm build:ios        # Build iOS archive
+pnpm lint             # Run Biome linter
+pnpm test             # Run unit tests
+pnpm test:e2e         # Run Maestro E2E tests
+```
+
+## Dependencies (Target)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)